app/controllers/fae/utilities_controller.rb

```diff
@@ -54,7 +54,25 @@ def records_by_display_name(query)
       all_models.each do |m|
         records += m.fae_search(query)
       end
-      records
+      authorize_records(records)
+    end
+
+    def authorize_records(records)
```

Comment:
@jasonfine I have some performance concerns with this method since we're looking up the authorization from the model per record. Since we're already looping through models via `all_models`, what if we filtered that down to `authorized_models`? I think that would drastically cut down on the `Fae::Authorization.access_map` look ups. What do you think?